views/partials/hibp_attribution.hbs

```diff
@@ -1,3 +1,3 @@
-<span class="source-info">
+<span class="source-info {{ class }}">
```

Comment:
Will this be confusing later that `source-info` is a hard-coded class for this attribute, and then `class` argument only adds to that - i.e., it doesn't replace the entire `class` property value?

Would it be clearer to do something like  `{{> hibp_attribution additionalClasses="align-left"`}}` and `<span class="source-info {{ additionalClasses }}">`

Reply:
Makes sense! Updated in the name of walking The Path of Most Future Sanity.